### ISSUE TITLE
Document list of supported file types

### ISSUE THREAD

**Issue by oxr463** [infinite-omicron/pii-redactor#3]
Programs:
- [x] RKE config

Scripts:
- [ ] Rancher2 Log Collector
  - [ ] `.log`
  - [ ] `.crt`

**Comment by oxr463** [comment]
For the Log Collector output, extract the archive it generates and then check all of file extensions.

```
tree | awk '{ split($0,a,"."); print a[2] }'
```

**Comment by outzhu** [comment]
that awk code prints out in the terminal blanks with a few lines of "pem, conflist, txt, log, 1-7" so I used grep to find lab2, ubuntu and 137.184.84.136:2379.

**Comment by oxr463** [comment]
> that awk code prints out in the terminal blanks with a few lines of "pem, conflist, txt, log, 1-7"

Yeah, the few lines it spit out are file extentions, e.g., PEM keys

> so I used grep to find lab2, ubuntu and 137.184.84.136:2379.

We have a hostname, an IPv4 address, and a port. What was that `ubuntu` reference from? It's fine to have that.

**Comment by outzhu** [comment]
21.1-19-gbad84ad4-0ubuntu1~20.04.2
/usr/lib/ubuntu-advantage/apt-esm-hook process-templates

most of it was 'ubuntu-s-4vcpu-8gb-amd-sfo3-01' (I think that was the main name of the droplet)

**Comment by oxr463** [comment]
> 21.1-19-gbad84ad4-0ubuntu1~20.04.2 /usr/lib/ubuntu-advantage/apt-esm-hook process-templates
> 
> most of it was 'ubuntu-s-4vcpu-8gb-amd-sfo3-01' (I think that was the main name of the droplet)

Ah okay. Yeah, that's not PII.
